chart/compass/charts/director/scopes.yaml

```diff
@@ -1,2 +1,39 @@
+query:
+  applications: ["application_read"]
```

Comment:
Hmm, I quickly checked naming conventions for scopes:
https://api.slack.com/docs/oauth-scopes
https://caseysoftware.com/blog/oauth-2-0-scopes-a-thought-experiment

And looks like colon is used widely. I think it would improve readability of the scopes. And, then we will be able to use underscore in other scopes, like `integration_system:read`